src/server_manager/ui_components/outline-server-settings.html

```diff
@@ -133,7 +161,11 @@ <h3>DigitalOcean</h3>
             <h3>[[localize('settings-server-info')]]</h3>
             <paper-input id="serverNameInput" class="server-name" value="{{name}}" label="[[localize('settings-server-name')]]" always-float-label maxlength="100" on-keydown="_handleNameInputKeyDown" on-blur="_handleNameInputBlur"></paper-input>
             <p class="detail">[[localize('settings-server-rename')]]</p>
-            <paper-input readonly value="[[serverPortForNewAccessKeys]]" label="[[localize('settings-access-key-port')]]" hidden$="[[!serverPortForNewAccessKeys]]" always-float-label maxlength="100"></paper-input>
+            <div id="portForNewAccessKeysContainer">
+              <paper-input id="keysPortInput" value="[[serverPortForNewAccessKeys]]" label="[[localize('settings-access-key-port')]]" maxlength="5" error-color="#ffffff" error-message="ERRORMESSSAGE" hidden$="[[!serverPortForNewAccessKeys]]" on-click="_handleKeysPortClick" on-keyup="_handleKeysPortInputKeyDown" always-float-label maxlength="100"></paper-input>
```

Comment:
Maybe you can style `error-color` with CSS?

Reply:
It was already set above with `--paper-input-container-invalid-color`.  Removed the extra noise.

Reply:
nit: style the underline with primary green.

<img width="340" alt="Screen Shot 2019-10-08 at 10 57 34 AM" src="https://user-images.githubusercontent.com/2132122/66407538-5ffbbb80-e9bb-11e9-9f2a-c62ae805b9a2.png">

Reply:
Done